kpatch-build/kpatch-build

```diff
@@ -540,13 +540,14 @@ usage() {
 	echo "		                        (can be specified multiple times)" >&2
 	echo "		-e, --oot-module        Enable patching out-of-tree module," >&2
 	echo "		                        specify current version of module" >&2
+	echo "		-p, --oot-module-src    Specify out-of-tree module source directory" >&2
```

Comment:
Will -p conflict with #1231 ?   (FWIW, I'd say -p for --profile-data is more intuitive... For the OOT options maybe we just drop the short options?  What do you think?)

Reply:
Agreed, I wasn't sure about it either. Its just every other option has a short version but I guess you need to start somewhere. I'll remove short versions from both oot-module-* options.

Reply:
Update looks good, thanks.